app/models/comment.rb

```diff
@@ -5,4 +5,10 @@ class Comment < ApplicationRecord
   def get_resolve
     self.statuses = :resolved
   end
+
+  def set_resolve; end
+
+  def get_author
+    self.author
```

Comment:
Style/RedundantSelf: Redundant `self` detected.

```suggestion
    author
```